src/pudl/etl/glue_assets.py

```diff
@@ -18,11 +17,14 @@
     outs={
         table_name: AssetOut(io_manager_key="pudl_sqlite_io_manager")
         for table_name in Package.get_etl_group_tables("glue")
+        # there may be a better way to restructure how we are making the glue post
+        # converting epacamd_eia -> an asset
+        if table_name != "epacamd_eia"
```

Comment:
yeah... I'd love your option about organization here. rn this new table and the old crosswalk is labeled as glue tables, but the code now all live over in the `etl.epacems_assets` module. but the metadata live in glue. Should I just reclassify all of these tables as cems tables? I feel swirly about it so I left it as is before getting more feedback

Reply:
It still seems like glue. I forget, was there an issue with creating these new assets in the `etl.glue_assets` module? You'll still have to filter out `epacamd_eia` in this line but will keep the metadata and assets in glue world.

Reply:
There are I think three distinctions here: where the code lives, what the dagster asset group is and what the metadata/etl-group is. these all feels swirly to me rn.

* rn code lives: in `etl.epacems_assets` bc the `clean_epacamd_eia` asset is required for `hourly_emissions_epacems` THEN both `clean_epacamd_eia` and `hourly_emissions_epacems` are required to make this `epacamd_eia_subplant_ids` asset. bc the assets are dependencies keeping all of these epacamd_eia assets in a gluey place caused circular dependencies. I thiiiiink I could break it out into three modules instead of the primary attempted 2 and current 1:
  * a clean_epacamd_eia place (should this be `etl.glue_assets` or `pudl.glue.epacamd_eia`?)
  * epacems_assets
  * a epacamd_eia_subplant_ids place (ditto to `etl` vs `glue` directory question
* dagster asset group: rn the glue assets are being derived from the etl-group and the epacamd assets are being derived from the assets in the `epacems_assets` module I believe.
* metadata/etl-group: rn I left these epacamd_eia tables in the glue etl group. bc they do feel like glue to me!

lol this is a lot and I'd love your perspective on it!! haaalp

Reply:
Yes, those are the three distinctions we need to work out generally. Hopefully, we clarify these distinctions when we tackle #2517. 

I feel like these are the ideal categorizations:
- `clean_epacamd_eia`, `epacamd_eia_subplant_ids `
   - etl_group: glue
   - asset_group: glue
   - code location: `etl.glue_assets`
- `hourly_emissions_epacems`
   - etl_group: epacems
   - asset_group: epacems
   - code location: `etl.epacems_assets`

I'll spend 15 - 20 min moving these around. If I can't figure it out, maybe we just keep everything in `etl.epacems_assets` and the epacems etl_group? Not ideal but I expect the code structure and etl groups to change in the next few sprints.

Reply:
I was able to move `raw_epacamd_eia`, `clean_epacamd_eia`, `emissions_unit_ids_epacems`, `epacamd_eia_subplant_ids` into `etl.glue_assets` without any circular imports. I think your little hack of filtering out `epacamd_eia` of the `create_glue_tables` multi_asset is fine for now.

Reply:
Okay cool! Wild. Now I'm confused as to why I was having circular import problems having both `clean_epacamd_eia` and `epacamd_eia_subplant_ids` in the same module. but alas this does seem to work all in the same place 🎉

Reply:
I've converted this one little line from `if table_name != "epacamd_eia"` to `if "epacamd_eia" not in table_name` so this now excludes the two epacamd_eia tables from this multi-asset.

Reply:
Sounds good to me!